### DISCUSSION TITLE
exporting db but as a x% sample of all tables

### DISCUSSION THREAD

**Discussion by sdmcallister** [duckdb/duckdb#3297] [General]
I have a 60+ gig db file and I want to get a local copy but as a sample (such as 10%) of all the tables to reduce the size.  I was hoping there might be an option during EXPORT but I don't see one. 

I am running this only myself so I suppose I could create a script and inject the table name into the sample query.

**Comment by hannes** [comment]
See here for example: https://duckdb.org/docs/sql/samples

**Comment by Alex-Monahan** [comment]
You could combine what Hannes sent above with the `COPY TO` statement to export a sample of each table individually!
[https://duckdb.org/docs/data/parquet](https://duckdb.org/docs/data/parquet)
